route53_static.tf

```diff
@@ -75,8 +91,19 @@ resource "aws_route53_record" "_amazonses_TXT" {
   provider = aws.route53resourcechange
 
   name    = "_amazonses.${aws_route53_zone.cyber_dhs_gov.name}"
-  records = ["tOxXTap6jGLn6/VnBnget7lrXW+TxZTyTdOhm8LbM/Y="]
+  records = ["${aws_ses_domain_identity.cyhy_dhs_gov_identity.verification_token}"]
   ttl     = 60
   type    = "TXT"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
+
+resource "aws_route53_record" "dkim_CNAME" {
+  provider = aws.route53resourcechange
+
+  count   = 3
+  name    = "${element(aws_ses_domain_dkim.cyber_dhs_gov_dkim.dkim_tokens, count.index)}._domainkey.${aws_route53_zone.cyber_dhs_gov.name}"
```

Comment:
See previous comment regarding the use of `element()`, preferring `for_each`, and the magic number `3`.